wgsl/index.bs

```diff
@@ -173,6 +173,12 @@ determined only by examining the program source.
 of these in the form of arrays and structures.
 Additional types describe memory.
 
+[SHORTNAME] does not have implicit conversions or promotions between numeric types.
+Converting a value from one numeric type to another requires
```

Comment:
nit: should this be numeric or boolean type instead?

Reply:
Fixed.